src/lib/osUtils/applescript.ts

```diff
@@ -1,19 +1,19 @@
 
 import * as shell from "@tauri-apps/api/shell"
-
+interface ReturnData { stdout: string[], stderr: string[] }
 export async function execString(command: string) {
```

Comment:
This function has no return type, does it really compile? It shouldn't, every standalone function must have type specified explicitly. If it compiler doesn't complain, it should be tweaked to, probably with `noImplicitReturns: true` in tsconfig.json.

Reply:
Sure I can clean it up more.